File: cpp/3_compound_intrest.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main(){
    
    double principle = 1000;       // amount they have bought

    double intrest = 5;         // it is %

    int time = 2;               // time in year or any time

    double amount = principle * pow( 1 + intrest/100,time) ;

    double compund_intrest = amount - principle;

    cout<<"The CI = "<<compund_intrest;

}

File: _3_compound_intrest.java
public class _3_compound_intrest{
    public static void main(String[] args){

        double principle = 1000;
        double intrest = 5;
        double time = 2;

        double amount = principle * Math.pow(1 + intrest / 100 , time);

        double CI = amount - principle;

        System.out.println(CI);

    }
}